# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1389036086 18000
#      Mon Jan 06 14:21:26 2014 -0500
# Branch gui-release
# Node ID 9a0f996a358ad53ca95818ac69f76170c3f76a97
# Parent  03226f21807734e590ebefda3134a3a3be426fda
maint: backout cset 42e6cdc76f4b on gui-release branch.

diff --git a/libinterp/corefcn/max.cc b/libinterp/corefcn/max.cc
--- a/libinterp/corefcn/max.cc
+++ b/libinterp/corefcn/max.cc
@@ -555,20 +555,16 @@ maximum value(s).  Thus,\n\
 %! assert (min (x, [], 1), reshape ([1, 3, 5, 7], [1,2,2]));
 %! assert (min (x, [], 2), reshape ([1, 2, 5, 6], [2,1,2]));
 %! [y, i] = min (x, [], 3);
 %! assert (ndims(y), 2);
 %! assert (y, [1, 3; 2, 4]);
 %! assert (ndims(i), 2);
 %! assert (i, [1, 1; 1, 1]);
 
-## Test for bug #40743
-%!assert (max (zeros (1,0), ones (1,1)), zeros (1,0))
-%!assert (max (sparse (zeros (1,0)), sparse (ones (1,1))), sparse (zeros (1,0)))
-
 %!error max ()
 %!error max (1, 2, 3, 4)
 */
 
 template <class ArrayType>
 static octave_value_list
 do_cumminmax_red_op (const octave_value& arg,
                      int nargout, int dim, bool ismin)
